front/src/modules/companies/utils/mapBoardFieldDefinitionsToViewFields.ts

```diff
@@ -0,0 +1,18 @@
+import { BoardFieldDefinition } from '@/ui/layout/board/types/BoardFieldDefinition';
+import { FieldMetadata } from '@/ui/object/field/types/FieldMetadata';
```

Comment:
this is not in the right folder but will be refactored with board later this week, LGTM :)